examples/js/nuxtjs/README.md

```diff
@@ -41,3 +41,34 @@ npm run build
 npm start
 ```
 __Note__: remember to replace the API keys in [`nuxt.config.js`](nuxt.config.js) with your own!
+
+## BugsnagSourceMapUploaderPlugin
+
+Example configuration of the `BugsnagSourceMapUploaderPlugin` in `nuxt.config.js`:
+
+```
+build: {
+	/*
+	** You can extend webpack config here
+	*/
+
+	extend ( config, { isDev, isClient } ) {
```

Comment:
```suggestion
	extend (config, { isDev, isClient }) {
```

Reply:
Thanks @bengourley. Yes, this was taken from a working customer example.